primitives/io/src/batch_verifier.rs

```diff
@@ -107,9 +107,30 @@ impl BatchVerifier {
 	) -> bool {
 		if self.invalid.load(AtomicOrdering::Relaxed) { return false; }
 		self.sr25519_items.push(Sr25519BatchItem { signature, pub_key, message });
+
+		if self.sr25519_items.len() >= 128 {
```

Comment:
Where is this number coming from?

Reply:
I noticed it is linear after 100tx-block

Reply:
@burdges ^^